### PR TITLE
Ad/linq/improve member expression error #719

### PR DESCRIPTION
**Pull request by AndyDentFree**

### CODE REVIEW

File: Tests/IntegrationTests.Shared/SimpleLINQtests.cs [hunk 1]
```diff
@@ -40,16 +41,16 @@ public void CreateList()
         {
             var s0 = _realm.All<Person>().Where(p => p.Score == 42.42f).ToList();
             Assert.That(s0.Count(), Is.EqualTo(1));
-            Assert.That(s0[0].Score, Is.EqualTo(42.42f));
+            Assert.That(s0 [0].Score, Is.EqualTo(42.42f));
```
Comment (kristiandupont) [inline Tests/IntegrationTests.Shared/SimpleLINQtests.cs]: Is this a Xamarin Studio formatting convention? That looks unusual to me..

**Comment by kristiandupont** [comment]
Retest this please

File: Tests/IntegrationTests.Shared/SimpleLINQtests.cs (hunk 1 above)
Comment (AndyDentFree) [inline Tests/IntegrationTests.Shared/SimpleLINQtests.cs]: Sigh, yes it looks like it crept back in at some point. I swear IDE settings randomly reset to default.

**Comment by kristiandupont** [comment]
:+1: I think the formatting changes should be reverted but it's not crucial.

**Pull request merged**